keyboards/smart/smart.c

```diff
@@ -0,0 +1,25 @@
+/* Copyright 2022 Jacky
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+ #include "quantum.h"
+
+//void bootloader_jump(void) {
```

Comment:
Previously commented out code has been asked to be removed. I wonder what should happen to these new changes....?

Reply:
I just removed.  Thanks

Reply:
I mean commented, I need to remove ?

Reply:
You shouldnt really resolve a conversation without doing anything...

Reply:
I delete the file.